README.md

```diff
@@ -4,226 +4,35 @@
-choose `Create and add user key` option. This should grant CircleCI privileges
-to push to the repository `https://github.com/USERNAME/DOC_REPO/`.
-
-If all goes well, you should be able to visit the documentation of your project
-on
+To install the module execute:
 ```
-https://github.com/USERNAME/DOC_REPO/DOC_URL
+pip install sklforestci
 ```
-
-### 7. Adding Badges
-
-Follow the instructions to add a [Travis Badge](https://docs.travis-ci.com/user/status-images/),
-[Coveralls Badge](https://coveralls.io) and
-[CircleCI Badge](https://circleci.com/docs/status-badges) to your repository's
-`README`.
-
-### 8. Advertising your package
-
-Once your work is mature enough for the general public to use it, you should
-submit a Pull Request to modify scikit-learn's
-[related projects listing](https://github.com/scikit-learn/scikit-learn/edit/master/doc/related_projects.rst).
-Please insert brief description of your project and a link to its code
-repository or PyPI page.
-You may also wish to announce your work on the
-[`scikit-learn-general` mailing list](https://lists.sourceforge.net/lists/listinfo/scikit-learn-general).
-
-### 9. Uploading your package to PyPI
-
-Uploading your package to [PyPI](https://pypi.python.org/pypi) allows users to
-install your package through `pip`. Python provides two repositories to upload
-your packages. The [PyPI Test](https://testpypi.python.org/pypi) repository,
-which is to be used for testing packages before their release, and the
-[PyPI](https://pypi.python.org/pypi) repository, where you can make your
-releases. You need to register a username and password with both these sites.
-The username and passwords for both these sites need not be the same. To upload
-your package through the command line, you need to store your username and
-password in a file called `.pypirc` in your `$HOME` directory with the
-following format.
-
+or
 ```shell
-[distutils]
-index-servers =
-  pypi
-  pypitest
-
-[pypi]
-repository=https://pypi.python.org/pypi
-username=<your-pypi-username>
-password=<your-pypi-passowrd>
-
-[pypitest]
-repository=https://testpypi.python.org/pypi
-username=<your-pypitest-username>
-password=<your-pypitest-passowrd>
-```
-Make sure that all details in `setup.py` are up to date. To upload your package
-to the Test server, execute:
-```
-python setup.py register -r pypitest
-python setup.py sdist upload -r pypitest
-```
-Your package should now be visible on: https://testpypi.python.org/pypi
-
-To install a package from the test server, execute:
-```
-pip install -i https://testpypi.python.org/pypi <package-name>
-```
-
-Similary, to upload your package to the PyPI server execute
-```
-python setup.py register -r pypi
-python setup.py sdist upload -r pypi
-```
-To install your package, execute:
-```
-pip install <package-name>
+$ python setup.py install
 ```
 
-*Thank you for cleanly contributing to the scikit-learn ecosystem!*
+## Example
+See examples gallery
```

Comment:
We should be able to link to the examples gallery from here, using the sphine :ref: directive. Something like what's described here: http://stackoverflow.com/questions/15394347/restructuredtext-adding-a-cross-reference-to-a-subheading-or-anchor-in-another